superset-frontend/plugins/legacy-preset-chart-nvd3/src/DistBar/index.js

```diff
@@ -35,7 +35,7 @@ const metadata = new ChartMetadata({
     { url: example2, caption: 'Grouped style' },
     { url: example3 },
   ],
-  name: t('Bar Chart'),
+  name: t('Bar Chart (legacy)'),
```

Comment:
I think it might make sense to make this conditional on the feature flag being flipped (`<feature flag check> ? 'Bar Chart (legacy)' : 'Bar Chart';`), as an env that's not using generic x-axis will not have an alternative to the legacy bar chart.

Reply:
Good idea, done!